src/common/_modules/application_gateway/main.tf

```diff
@@ -412,20 +412,29 @@ module "app_gw" {
       }
     }
 
-    ipatente-io-pagopa-it = {
+    vehicles-ipatente-io-pagopa-it = {
```

Comment:
Can we add this change in the next PR? Just to have a Terraform plan easier to read